File: level2/최댓값과_최솟값/solution.js
function solution(s) {
  let arr = s
    .split(" ")
    .map((v) => Number(v))
    .sort((a, b) => a - b);
  var answer = [arr[0], arr[arr.length - 1]].join(" ");
  return answer;
}

console.log(solution("1 2 3 4"));
console.log(solution("-1 -2 -3 -4"));
console.log(solution("-1 -1"));

File: level2/최댓값과_최솟값/solution.rb
def solution(s)
  tmp = s.split(" ").sort { |a, b| a.to_i <=> b.to_i }
  return "#{tmp.first} #{tmp.last}"
end

# 다른 사람 풀이. Ruby에는 신박한 메소드가 참 많다는 걸 계속 느낀다
def solution2(s)
  s.split.map(&:to_i).minmax.join(" ")
end

p solution "1 2 3 4"
p solution "-1 -2 -3 -4"
p solution "-1 -1"
